packages/libs/react-ui/package.json

```diff
@@ -71,6 +71,7 @@
     "@types/node": "^16.0.0",
     "@types/react": "^18.0.30",
     "@vanilla-extract/webpack-plugin": "~2.2.0",
+    "classnames": "^2.3.1",
```

Comment:
We should remove this from devDependencies since it needs to be a normal dependency